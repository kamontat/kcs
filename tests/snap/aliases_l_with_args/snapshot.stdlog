00:10:45 [INF] private.base.settings : testing mode has been enabled
00:10:45 [DBG] libs.functions.loader.run : run '__kcs_tmp_hook_setup' function with 0 args []
00:10:45 [DBG] libs.functions.loader.run : run '__kcs_l_hook_setup' function with 0 args []
00:10:45 [DBG] libs.commands.loader.shell : run '$KCT_PATH_TESTDIR/commands/aliases/looooooong.sh' using 'bash' with 3 args [a b c]
00:10:45 [DBG] libs.functions.loader.run : run '__kcs_tmp_hook_setup' function with 0 args []
00:10:45 [DBG] libs.functions.loader.run : run '__kcs_long_hook_main' function with 4 args [cmd.long a b c]
00:10:45 [DBG] libs.functions.loader.run : run '__kcs_environment_hook_clean' function with 0 args []
00:10:45 [DBG] libs.functions.loader.run : run '__kcs_configs_hook_clean' function with 0 args []
00:10:45 [DBG] libs.functions.loader.run : run '__kcs_tmp_hook_clean' function with 0 args []
00:10:45 [DBG] libs.functions.loader.run : run '__kcs_log_hook_clean' function with 0 args []
00:10:45 [DBG] libs.functions.loader.run : run '__kcs_environment_hook_clean' function with 0 args []
00:10:45 [DBG] libs.functions.loader.run : run '__kcs_configs_hook_clean' function with 0 args []
00:10:45 [DBG] libs.functions.loader.run : run '__kcs_tmp_hook_clean' function with 0 args []
00:10:45 [DBG] libs.functions.loader.run : run '__kcs_log_hook_clean' function with 0 args []
