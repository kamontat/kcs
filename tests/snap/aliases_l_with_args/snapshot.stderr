00:10:45 [ERR] private.base.exit : loading 'func:__kcs_long_hook_main' failed
00:10:45 [ERR] private.base.exit : loading 'commands:aliases/looooooong' failed ($KCT_PATH_TESTDIR/commands/aliases/looooooong.sh)
00:10:45 [ERR] private.base.exit : loading 'func:__kcs_l_hook_setup' failed
00:10:45 [ERR] private.base.exit : loading 'commands:aliases/l' failed ($KCT_PATH_TESTDIR/commands/aliases/l.sh)
